layer/utils/grpc/channel.py

```diff
@@ -38,7 +38,7 @@ def create_grpc_channel(
                     "retryPolicy": {
                         "maxAttempts": 5,
                         "initialBackoff": "0.1s",
-                        "maxBackoff": "5s",
+                        "maxBackoff": "30s",
```

Comment:
I think you should increase the attempts not the backoff? I don't think we ever reach that now anyway

Reply:
We are going to do 0.1, 0.2, 0.4, 0.8, 1.6 and then exit with or without this change.

Reply:
You're right, fixing